baybe/searchspace/continuous.py

```diff
@@ -42,6 +44,36 @@ class SubspaceContinuous(SerialMixin):
     )
     """List of linear inequality constraints."""
 
+    def __repr__(self) -> str:
+        """Override the standard __repr__."""
+        if self.is_empty:
+            return ""
+        # Convert the lists to dataFrames to be able to use pretty_printing
+        par_df = parameter_cartesian_prod_to_df(self.parameters)
+        const_lin_eq_df = parameter_cartesian_prod_to_df(self.constraints_lin_eq)
+        const_lin_ineq_df = parameter_cartesian_prod_to_df(self.constraints_lin_ineq)
+
+        # Put all attributes of the continuous class in one string
+        continuous_str = (
+            "\n\n \033[1m |--> The continuous search space \n\n \033[0m"
```

Comment:
Haven't yet executed the code, but could you briefly explain what `\033[1m |-->` does?

Reply:
The expression `\033[1m` causes the output to be thicker, but the `|-->` is part of the output itself. 
The output looks like this: 
<img width="321" alt="thickerOutput" src="https://github.com/emdgroup/baybe/assets/118910837/ac6f14ae-4b47-4adc-bc72-8cd32d4f01c9">